src/nnbench/types.py

```diff
@@ -130,12 +126,49 @@ class LocalArtifactLoader(ArtifactLoader):
     """
 
     def __init__(self, path: str | os.PathLike[str]) -> None:
-        self._path = path
+        import hashlib
+
+        self._path = Path(path)
+        self._hash = hashlib.md5(usedforsecurity=False)
+
+    def verify(self, expected: str, blocksize: int = 2**22) -> None:
```

Comment:
Should we add the usage of `verify` to the guide (`/docs/guides/artifact.md`)?

I can also take care of that.

Reply:
We should in any case update the guide/examples when we make another round of changes to the artifact API.